.github/workflows/create-release.yml

```diff
@@ -7,80 +7,42 @@ permissions:
 on:
   workflow_dispatch:
   push:
-    branches: [ "main" ]
+    branches: ["main"]
     paths:
-      - 'Cargo.toml'
+      - "Cargo.toml"
 
 jobs:
-  # Build and packages all the things
-  build-binaries:
-    if: |
-      contains(github.event.head_commit.message, 'chore: Release') == 'true' || \
-      contains(github.event.head_commit.message, 'chore: Bump') == 'true' || \
-      github.event_name == 'workflow_dispatch'
-    strategy:
-      matrix:
-        # For these target platforms
-        include:
-          - target: x86_64-unknown-linux-gnu
-            os: ubuntu-latest
-            arch: x86_64
-            ext: deb
-            protoc-arch: linux-x86_64
-          - target: x86_64-apple-darwin
-            os: macos-latest
-            arch: x86_64
-            ext: pkg
-            protoc-arch: osx-x86_64
-          - target: aarch64-apple-darwin
-            os: macos-latest
-            config-file: fpm/osx.fpm
-            arch: aarch64
-            ext: pkg
-            protoc-arch: osx-aarch_64
-        #- target: x86_64-pc-windows-msvc
-        #  os: windows-latest
-    runs-on: ${{ matrix.os }}
+  release:
+    name: Create Release
+    runs-on: ubuntu-latest
     env:
-      GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Do we even need this if we're not publishing a GH release?

Reply:
Added binaries back